README.md

```diff
@@ -1,46 +1,119 @@
 # react-maybe
+
 Render one of two React components
 
 [![Build Status](https://travis-ci.org/strange-developer/react-maybe.svg?branch=master)](https://travis-ci.org/strange-developer/react-maybe)
 
-react-maybe is build behind a monad which conditionally displays one of two components based on the value contained in the `of` prop. If the value is one of `null`, `undefined` or `false`, the `orElse` component will be displayed. In all other cases the component passed to the `either` prop will be displayed.
+react-maybe is built behind a monad which conditionally displays one of two components based on the
+value contained in the `of` prop. If the value is one of `null`, `undefined` or `false`, the
+`orElse` component will be displayed. In all other cases the component passed to the `either` prop
+will be displayed.
+
+Additionally, the value in the `of` prop can be altered through the use of the `map` prop. The `map`
```

Comment:
I'm going to be that guy. Just thinking that we should remove the word `Additionally`. We could possibly mention that calling `map` will affect which component is returned. That may be taking it a bit too far. Up to you. Great work!!!

P.S
Thanks for adding the npm install link. Been meaning to add that as well.